SalesforceFoundation/EDA
# W-037247 - When new Contact is created, map Contact currency to Account Currency

# Critical Changes

# Changes
## Update to Default Currency Settings for Accounts
In orgs that have enabled multiple currencies, the currency setting for a newly created Contact that is not related to an existing Household or Administrative Account will now be copied to the currency setting on the Account that's created for the new Contact.

For important information about enabling multiple currencies in your org, see [Manage Multiple Currencies](https://help.salesforce.com/articleView?id=admin_currency.htm&type=5) and [Considerations for Enabling Multiple Currencies](https://help.salesforce.com/articleView?id=admin_enable_multicurrency_implications.htm&type=5).

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
- Go to Setup -> Company Information, click edit and check 'Activate Multiple Currencies'
- Go to Setup -> Manage currency, add EUR as a new currency type
- Go to EDA Setting page, change the default account model to 'Household'
- Add 'Contact currency' to HEDA contact page layout and 'Account currency' to HEDA Household page layout
- Create a new contact with currency = 'EUR'
- Make sure the currency of related account, aka test contact 'Household', is 'EUR' as well
